### PR TITLE
Docs: Add note about enqueueing assets in the iframe and trim whitespace.

### PR DESCRIPTION
**Pull request by ndiego**
## What?
Updates the [Enqueueing assets in the Editor](https://developer.wordpress.org/block-editor/how-to-guides/enqueueing-assets-in-the-editor/) doc based on @ellatrix's [feedback](https://github.com/WordPress/gutenberg/pull/53828#discussion_r1308693030). While updating the doc, I also trimmed all extra whitespace.

Edit: in preparing this PR, a number of typos were also found and fixed.

### CODE REVIEW

**Comment by alexstine** [comment]
Conflicts. 😞 Happy to review once resolved.

File: docs/how-to-guides/enqueueing-assets-in-the-editor.md [hunk 1]
```diff
@@ -29,8 +29,8 @@ function example_enqueue_editor_assets() {
         'example-editor-scripts', 
         plugins_url( 'editor-scripts.js', __FILE__ ) );
     }
```
Comment (alexstine) [inline docs/how-to-guides/enqueueing-assets-in-the-editor.md]: Why is this closing `}` here?

File: docs/how-to-guides/enqueueing-assets-in-the-editor.md [hunk 2]
```diff
@@ -29,8 +29,8 @@ function example_enqueue_editor_assets() {
...
-    wp_enqueue_style( 
-        'example-editor-styles', 
+    wp_enqueue_style(
+        'example-editor-styles',
         plugins_url( 'editor-styles.css', __FILE__ ) );
     }
```
Comment (alexstine) [inline docs/how-to-guides/enqueueing-assets-in-the-editor.md]: Why is this closing `}` here?

**Review by alexstine — COMMENTED** [review]
@ndiego In the `wp_enqueue_script` and `wp_enqueue_style` function calls, they are terminated via `);` but there is an existing `}` on the following line. I believe this will cause a syntax error when executing this function. The closing `}` for the function should be at the same indent level hints the last `}` before the `add_action` hook. The others should likely be removed.

File: docs/how-to-guides/enqueueing-assets-in-the-editor.md (hunk 2 above)
Comment (ndiego) [reply docs/how-to-guides/enqueueing-assets-in-the-editor.md]: Yeah, those were blatant typos 🤦‍♂️ All fixed.

**Review by alexstine — COMMENTED** [review]
@ndiego There is still a problem with this first code snippet.

```php
/**
 * Enqueue Editor assets.
 */
function example_enqueue_editor_assets() {
    wp_enqueue_script(
        'example-editor-scripts',
        plugins_url( 'editor-scripts.js', __FILE__ ) );
    }
    wp_enqueue_style(
        'example-editor-styles',
        plugins_url( 'editor-styles.css', __FILE__ ) );
    }
}
add_action( 'enqueue_block_editor_assets, 'example_enqueue_editor_assets' );
```

Rewrite to:

```php
/**
 * Enqueue Editor assets.
 */
function example_enqueue_editor_assets() {
    wp_enqueue_script(
        'example-editor-scripts',
        plugins_url( 'editor-scripts.js', __FILE__ )
    );
    wp_enqueue_style(
        'example-editor-styles',
        plugins_url( 'editor-styles.css', __FILE__ )
    );
}
add_action( 'enqueue_block_editor_assets, 'example_enqueue_editor_assets' );
```

**Comment by ndiego** [comment]
Thanks @alexstine, and there was a missing `'` in the `add_action`. All fixed. I also updated the description of this PR to reflect these changes 😅

**Review by alexstine — APPROVED** [review]
👍

**Pull request merged**